### DISCUSSION TITLE
Is it possible to set the water level through the card?

### DISCUSSION THREAD

**Discussion by lelukas** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#557] [Q&A]
I only see controls for controlling the power, both on the vaccum device and the card itself. Does the integration expose ways for controlling the water level?

**Comment by PiotrMachowski** [comment]
This is not an integration, this is just a Lovelace card. If your integration supports controlling the water level then the card can be configured to do it as well (e.g. #341)
